server/webapp/WEB-INF/rails.new/app/controllers/admin/status_reports_controller.rb

```diff
@@ -32,16 +32,27 @@ def show
       render_plugin_error e
     end
 
+    def show
+      @view_title = 'Plugin Status Report'
```

Comment:
Can we change it to `Agent status report(agentid)` ?

Reply:
Also, should the actions be `agent_status` and `plugin_status` over `index` and  `show`?